cmd/meroxa/turbine/golang/deploy.go

```diff
@@ -52,13 +50,11 @@ func (t *turbineGoCLI) Deploy(ctx context.Context, imageName, appName, gitSha, s
 		return deploymentSpec, errors.New(string(output))
 	}
 
-	if specVersion != "" {
-		deploymentSpec, err = utils.GetTurbineResponseFromOutput(string(output))
-		if err != nil {
-			err = fmt.Errorf(
-				"unable to receive the deployment spec for the Meroxa Application at %s", t.appPath)
-		}
+	deploymentSpec, err = utils.GetTurbineResponseFromOutput(string(output))
+	if err != nil {
+		err = fmt.Errorf("unable to receive the deployment spec for the Meroxa Application at %s", t.appPath)
 	}
+
 	return deploymentSpec, err
 }
```

Comment:
I believe you can also remove this branching point:
https://github.com/meroxa/cli/blob/c4e2eb0acff3d9db3dbadbc882c9961c95e3f693/cmd/meroxa/turbine/golang/deploy.go#L119-L123

It was there before to support the old cli

Reply:
@dianadoherty this is for building which is something unrelated to IR deployments. I think this is still relevant.

Reply:
Update: this change was technically added unrelated to IR (https://github.com/meroxa/turbine-go/pull/88). We should still support this way of parsing turbine's library output for those running older versions of turbine.